### PR TITLE
Feature/robots

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/data.gov/issues/3648

Very WIP still

### CODE REVIEW

File: docker-entrypoint.d/10-setup-db.sh [hunk 1]
```diff
@@ -1,5 +1,12 @@
+#!/bin/bash
+
+echo "Installing updated cryptography.."
+pip install cryptography==37.0.4
+pip list
```
Comment (jbrown-xentity) [inline docker-entrypoint.d/10-setup-db.sh:5]: Remove the pip commands here

File: .env [hunk 2]
```diff
@@ -71,3 +73,19 @@ CKAN___REDIS_HOST=redis
...
+CKAN_STORAGE_PATH=/var/lib/ckan/files
+CKANEXT__S3SITEMAP__AWS_ACCESS_KEY_ID=_placeholder
+CKANEXT__S3SITEMAP__AWS_BUCKET_NAME=catalog_sitemap
+CKANEXT__S3SITEMAP__AWS_S3_URL=_placeholder
+CKANEXT__S3SITEMAP__AWS_SECRET_ACCESS_KEY=_placeholder
+CKANEXT__S3SITEMAP__AWS_STORAGE_PATH=local
+CKANEXT__S3SITEMAP__REGION_NAME=us-east-1
+CKANEXT__S3SITEMAP__HOST_NAME=http://localstack-container:4566
+CKANEXT__S3SITEMAP__PUBLIC_HOST_NAME=http://localhost:4566
+CKANEXT__S3SITEMAP__LOCALSTACK_ENDPOINT=http://localhost:4566
```
Comment (jbrown-xentity) [inline .env:89]: Is this still needed, specifically adding the localstack endpoint as a variable?

**Review by jbrown-xentity — APPROVED** [review]
Minor changes, but glad we got past the build blockers!

File: docker-entrypoint.d/10-setup-db.sh (hunk 1 above)
Comment (robert-bryson) [reply docker-entrypoint.d/10-setup-db.sh]: Good catch, thanks.

File: .env (hunk 2 above)
Comment (robert-bryson) [reply .env:89]: Hmm, good point. I'll investigate a bit and should be able to pull it out.

Comment (robert-bryson) [reply .env:89]: I played around with a couple options for how best to inject the localstack URL, would like to discuss later. Merging for now to check behavior in dev.

**Pull request merged**